plugin/PDFView.qml

```diff
@@ -278,29 +292,29 @@ SilicaFlickable {
 
         property bool _pageSizesReady
 
-        width: base.width
-
+        y: header.height
+        width: root.width
         spacing: Theme.paddingLarge
-        flickable: base
+        flickable: root
         linkWiggle: Theme.itemSizeMedium / 2
         linkColor: Theme.highlightColor
-        pagePlaceholderColor: Theme.highlightColor
+        pagePlaceholderColor: "white"
```

Comment:
I think I liked the older version better myself, at least on light ambience it nicely shows where the pages start and stop when they are not yet loaded. For another detail the placeholder transparency is hard-coded in canvas.

Reply:
The highlight color is often much more saturated than background, and the document contents are unlikely to be colored in highlight color. Documents are predominantly white, thus white placeholder minimizes flicker and calms down the scrolling.